leshan.py

```diff
@@ -105,13 +105,20 @@ def run(client, url, hostname, port, monitor):
         response = get(client_list_url, raw=True)
         if response:
             for target in response:
+		Update = True
```

Comment:
PEP8 complains about these being upper case... "Use the function and variable naming rules: lowercase with words separated by underscores as necessary to improve readability."